src/main/java/org/candlepin/subscriptions/ApplicationProperties.java

```diff
@@ -43,6 +43,9 @@
 
   private boolean prettyPrintJson = false;
 
+  /** Fail to start the application is a FileSource is missing */
+  private boolean strictResourceLoadingMode = true;
+
```

Comment:
It's part of the StructuredFileSource refactor.  I put it in there with a default of true but I like having the options so developers can set it to false in config or for testing.  I've run into test issues before with the test failing because some config file can't be loaded and I end up having to pass in a dummy.

Reply:
@awood, yeah, but I think it got superseded by `strictProfileLoadingMode` in `ProductProfileProperties`? I tried both `git grep -i strictresourceloadingmode` and `Find Usages` in intellij and found no references... Am I missing something?

Reply:
> @awood, yeah, but I think it got superseded by `strictProfileLoadingMode` in `ProductProfileProperties`? I tried both `git grep -i strictresourceloadingmode` and `Find Usages` in intellij and found no references... Am I missing something?

Aha!  I see.  I removed it.